File: made_in_python/bubble_sort.py
from random import randint
from time import time

the_list = [0] * 55555

for i in range(len(the_list)): # Python's integer is 64 bit. Becuz of that I made the randomizer like this.
    the_list[i] = randint(-2147483648, 2147483647 - 1) # -1 becuz the endpoint is included

def sort(number_list):
    """ In Bubble Sort, the program goes through the array and checks the elements and the elements next to
    it. If the left element that program is checking is bigger than the right next element, the program
    swaps the elements. When the array ends, if the array is not sorted, the program repeats this process
    until the array is sorted. """

    i: int
    j: int

    while True:
        i = 0

        for j in range(1, len(number_list)):
            if number_list[j - 1] > number_list[j]:
                number_list[j - 1], number_list[j] = number_list[j], number_list[j - 1] # Swapping elements
                i += 1

        if i == 0:
            break

start_time: float = time()
sort(the_list)
end_time: float = time()
#print("[" + ", ".join(str(x) for x in the_list) + "]") # To see the array
print((str)(len(the_list)) + " integers sorted in " + (str)((end_time - start_time) * 1000) + " milliseconds.")

File: MadeInJava/src/BubbleSort.java
import java.util.Random;
//import java.util.Arrays; // Use this too if you want to see the array

public class BubbleSort {
    public static void main(String[] args) {
        int[] theArray = new int[120000];
        Random r = new Random();

        for (int i = 0; i < theArray.length; i++)
        {
            theArray[i] = r.nextInt();
        }

        long startTime = System.nanoTime();
        Sort(theArray);
        long endTime = System.nanoTime();
        //System.out.println(Arrays.toString(theArray)); // To see the array
        System.out.println(theArray.length + " integers sorted in " + ((endTime - startTime) / 1000000.0) + " milliseconds");
    }

    static void Sort(int[] numberArray)
    {
        /* In Bubble Sort, the program goes through the array and checks the elements and the elements next to
        it. If the left element that program is checking is bigger than the right next element, the program
        swaps the elements. When the array ends, if the array is not sorted, the program repeats this process
        until the array is sorted. */

        int i, j, temp;

        do
        {
            i = 0;

            for (j = 1; j < numberArray.length; j++)
            {
                if (numberArray[j - 1] > numberArray[j])
                {
                    temp = numberArray[j - 1];
                    numberArray[j - 1] = numberArray[j];
                    numberArray[j] = temp;
                    i++;
                }
            }
        }
        while (i > 0);
    }
}
